cmd/build.go

```diff
@@ -12,7 +12,6 @@ func NewBuildCmd(f factory.Factory, globalFlags *flags.GlobalFlags, rawConfig *R
 	cmd := &RunPipelineCmd{
 		GlobalFlags:             globalFlags,
 		Pipeline:                "build",
-		ForceBuild:              true,
```

Comment:
Hello! This is only used as the default value, and can be changed to false by passing `--force-build=false`. The code that allows the flag value to be used is [this call](https://github.com/devspace-sh/devspace/blob/main/cmd/build.go#L36), which [configures cobra to use the passed flag](https://github.com/devspace-sh/devspace/blob/main/cmd/run_pipeline.go#L75).